task2/User.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+include "../vendor/autoload.php";
+
+use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Component\Validator\Mapping\ClassMetadata;
+
+class User
+{
+    private int $id;
+    private string $name;
+    private string $email;
+    private string $password;
+    private DateTime $creationDateTime;
+
+    public function __construct($id, $name, $email, $password)
+    {
+        $this->id = $id;
+        $this->name = $name;
+        $this->email = $email;
+        $this->password = $password;
+        $this->creationDateTime = date('Y-m-d H:i:s');
```

Comment:
В описании свойства класса указан тип `DateTime`, а тут присваивается строка.
Лучше сразу сохранять объект класса DateTime:
```suggestion
        $this->creationDateTime = new DateTime();
```